server/src/service/CardService.js

```diff
@@ -185,4 +186,50 @@ export class CardService extends BaseService {
             })),
         };
     }
+
+    @Transactional()
+    async createCard({ userId, listId, title, dueDate, content }) {
+        const list = await this.listRepository
+            .createQueryBuilder('list')
+            .select(['list.id', 'list.board'])
+            .where('list.id = :listId', { listId })
+            .getRawOne();
+        if (!list) throw new EntityNotFoundError();
+        const boardService = BoardService.getInstance();
+        await boardService.checkForbidden(userId, list.board_id);
+
+        const cardWithMaxPosition = await this.cardRepository
+            .createQueryBuilder('card')
+            .select('MAX(card.position)', 'maxPosition')
+            .where('card.list = :listId', { listId })
+            .getRawOne();
+        const card = {
+            title,
+            content,
+            position: cardWithMaxPosition.maxPosition + 1,
+            dueDate,
+            list: list.list_id,
+            creator: userId,
+        };
+        const createCard = this.cardRepository.create(card);
+        await this.cardRepository.save(createCard);
```

Comment:
```suggestion
        const card = this.cardRepository.create({
            title,
            content,
            position: cardWithMaxPosition.maxPosition + 1,
            dueDate,
            list: list.list_id,
            creator: userId,
        });
        await this.cardRepository.save(card);
```
변수를 하나 줄일 수 있는 방향이 좋다고 생각합니다